test/router-luatest/router_2_2_test.lua

```diff
@@ -1233,7 +1236,7 @@ g.test_replicaset_services_do_not_spam_same_errors = function(g)
         ivshard.router.internal.static_router.replicasets[uuid].master = nil
     end, {g.replica_2_a:replicaset_uuid()})
     msg = "Error during master search"
-    g.router:wait_log_exactly_once(msg, {timeout = 0.1, on_yield = function()
+    g.router:wait_log_exactly_once(msg, {timeout = 0.5, on_yield = function()
```

Comment:
How increase of waiting timeout makes the test less flaky? `wait_log_exactly_once` waits for `timeout` seconds and constantly checks the logs, so that the same message doesn't appear. By making timeout greater we just prolong the time of tests and do not make it less flaky

The same comment for all other places